subcommands/devices/list.go

```diff
@@ -221,6 +226,8 @@ func doList(cmd *cobra.Command, args []string) {
 	}
 	dl, err := api.DeviceList(
 		deviceMine,
+		deviceOnlyProd,
+		deviceOnlyNonProd,
```

Comment:
Seems like this could be a single flag and simplify the function args.

Reply:
I've tried to play with `*bool` type of a flag, and it resulted into a mess.
Counter-intuitively, in Golang using two boolean flags is much easier...

Reply:
ah - so this is 3 cases. all devices, non-prod only, and prod-only. I guess this is probably fine, some type of enum might simplify, but we always think we are done adding options to this function